addons/website_slides/models/slide_channel.py

```diff
@@ -216,19 +225,20 @@ def _compute_slides_statistics(self):
             result[cid].update(cdata)
 
         for record in self:
-            record.update(result[record.id])
+            if record.id in result:
```

Comment:
Why this change ? result is build from self.ids so normally there should be no hole. Is it at create ?